specs/whereAll.test.ts

```diff
@@ -2,8 +2,30 @@ import { describe, it } from './_describe.ts'
 import { whereAll, curry } from '../mod.ts'
 import { eq } from './utils/utils.ts'
 
+//TODO: (shivam-singla) check this (undefined ones)
+
 describe('whereAll', () => {
   const equals = curry(2, (x: number, y: number) => x === y)
+
+  // @ts-ignore
```

Comment:
what is this comment doing here?

Reply:
There is a case which involves undefined.